slack-bot/tests/test_event_handlers.py

```
@@ -0,0 +1,222 @@
+        # Verify that event decorators were called
+        assert mock_app.event.call_count >= 2  # Should register multiple events
+
+    @pytest.mark.asyncio
+    async def test_assistant_thread_started_handler(self, mock_app, mock_slack_service, mock_llm_service):
+        """Test assistant thread started event handler"""
+        mock_slack_service.send_message = AsyncMock()
+        
+        await register_handlers(mock_app, mock_slack_service, mock_llm_service)
+        
+        # Simulate the handler being called directly
+        body = {
```

Ruff (0.11.9)

Local variable `body` is assigned to but never used

Remove assignment to unused variable `body`

(F841)